coordinator/src/bin/coordinator.rs

```diff
@@ -94,10 +94,9 @@ async fn main() -> Result<()> {
     let mut conn = pool.get()?;
     run_migration(&mut conn);
 
-    let node = Node {
-        inner: node,
-        pool: pool.clone(),
-    };
+    let settings = Settings::new().await;
+    let node = Node::new(node, pool.clone());
+    node.update_settings(settings.as_node_settings()).await;
```

Comment:
because it was a bit more troublesome, and I realised it's not worth it 😔

I prefer having a single way of updating the settings, as I found it less confusing.

However, to properly pull it off would need:
- to replicate the logic of applying the settings deeper (to the wallet) - can't just reuse the function in a constructor
- make the constructor async

it's certainly possible, I just wanted to keep a pattern consistent.